backend/typescript/models/camp.model.ts

```diff
@@ -7,13 +7,14 @@ export interface Camp extends Document {
   ageLower: number;
   ageUpper: number;
   capacity: number;
-  name: string;
+  campSessions: (Schema.Types.ObjectId | CampSession)[];
   description: string;
-  location: string;
   fee: number;
-  formQuestions: (Schema.Types.ObjectId | FormQuestion)[];
-  campSessions: (Schema.Types.ObjectId | CampSession)[];
   fileName?: string;
+  formQuestions: (Schema.Types.ObjectId | FormQuestion)[];
+  location: string;
+  name: string;
+  productId: string;
```

Comment:
Do we need productId in camp model if we are following `prod_<mongoose_id>` syntax for stripe products?

Reply:
Decided to let Stripe generate the id stored in productId